.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Proposed changes
+
+<!--
+Describe an overview of the changes, why they are useful, and any comments on the implementation so that it is easy for reviewers to understand the PR.
+-->
+
+### Types of changes:
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+
+### Component:
+
+- [ ] Code
+- [ ] Documentation
+- [ ] Build system
+- [ ] Continuous integration
+
+### Code review checklist
+
+- [ ] Follows [code review guidelines](https://sxs-collaboration.github.io/spectre/code_review_guide.html)
+- [ ] Correct exception specification
+- [ ] Mark objects `const` whenever possible
+- [ ] Almost always `auto`
+- [ ] Passes clang-tidy, CppCheck, and has no compiler warnings
+- [ ] All dependent changes have been merged
+- [ ] Code is tested
+- [ ] Code has documentation
```

Comment:
And documentation compiles without warnings (Run make doc 2>&1 | grep 'warning' to check)

Reply:
Could we just summarize this and several of your other points as "Clean Travis run" or similar?

Reply:
@wthrowe and I had an offline discussion proposing the idea that maybe the template should be where we actually list all the code review guidelines instead of referring people to another website. Thoughts on this? Once we settle on this I will make the changes to this block.